src/clipboard-minimal.js

```diff
@@ -0,0 +1,31 @@
+function clipboard() {
+  return {
+    copy: (input) => {
+      const isRTL = document.documentElement.getAttribute('dir') == 'rtl';
+
+      const fakeElem = document.createElement('textarea');
+      // Prevent zooming on iOS
+      fakeElem.style.fontSize = '12pt';
+      // Reset box model
+      fakeElem.style.border = '0';
+      fakeElem.style.padding = '0';
+      fakeElem.style.margin = '0';
+      // Move element out of screen horizontally
+      fakeElem.style.position = 'absolute';
+      fakeElem.style[ isRTL ? 'right' : 'left' ] = '-9999px';
+      // Move element to the same position vertically
+      let yPosition = window.pageYOffset || document.documentElement.scrollTop;
+      fakeElem.style.top = `${yPosition}px`;
+
+      fakeElem.setAttribute('readonly', '');
+      fakeElem.value = input;
```

Comment:
Maybe this part could be separated to a single method, something like `generateFakeElement`, what do you think about?

Reply:
also, another point here It does like duplication logic here, 	we have the same logic in the https://github.com/zenorocha/clipboard.js/blob/master/src/clipboard-action.js#L47 

I do believe that we can avoid that

Reply:
Have not the same behavior of the clipboard action on that case, we follow the same principle of other implementations used as reference.

Reply:
I agreed that we can at least wrap the code duplicated in a short helper. Just to improve maintainability. WDYT @ktfth @vitormalencar ?